Gemfile

```diff
@@ -4,19 +4,18 @@ ruby File.read(".ruby-version").strip
 
 source "https://rubygems.org"
 
-gem "rails", "~> 6.0.2"
-
 gem "asset_sync"
 gem "bootsnap", "~> 1"
-gem "dynamoid"
 gem "fog-aws"
 gem "govuk_app_config", "~> 2.1.1"
-gem "govuk_publishing_components", "~> 21.37.0"
+gem "govuk_publishing_components", "~> 21.38.0"
 gem "lograge"
+gem "pg", "~> 1"
```

Comment:
[See above](https://github.com/alphagov/govuk-coronavirus-triage-vulnerable-people/pull/1/commits/eb1d7887954bb2748a1e2c72e4fdcf9ac4721792#r403925671) might be able to remove